files/zh-tw/learn/javascript/first_steps/math/index.md

```diff
@@ -38,7 +38,7 @@ slug: Learn/JavaScript/First_steps/Math
 
 - **二進位** —計算機的最底層語言；0 和 1。
 - **八進位** —以 8 為單位，每列使用 0–7。
-- **十六進位** —以 16 為單位，在每列中使用 0–9，然後使用 a–f。[在 CSS 中](https://translate.googleusercontent.com/translate_c?depth=1&pto=aue&rurl=translate.google.com.tw&sl=en&sp=nmt4&tl=zh-TW&u=https://developer.mozilla.org/en-US/Learn/CSS/Introduction_to_CSS/Values_and_units&usg=ALkJrhjxtle73XLvao6e9e-CZv6u2Hq68g#Hexadecimal_values)設置[顏色](https://translate.googleusercontent.com/translate_c?depth=1&pto=aue&rurl=translate.google.com.tw&sl=en&sp=nmt4&tl=zh-TW&u=https://developer.mozilla.org/en-US/Learn/CSS/Introduction_to_CSS/Values_and_units&usg=ALkJrhjxtle73XLvao6e9e-CZv6u2Hq68g#Hexadecimal_values)之前，您可能已經遇到過這些數字。
+- **十六進位** —以 16 為單位，在每列中使用 0–9，然後使用 a–f。[在 CSS 中](/zh-TW/Learn/CSS/Introduction_to_CSS/Values_and_units#Hexadecimal_values)設置[顏色](/zh-TW/Learn/CSS/Introduction_to_CSS/Values_and_units#Hexadecimal_values)之前，您可能已經遇到過這些數字。
```

Comment:
Wasn't sure if there was a way to combine the 2 duplicate links, but I figured getting rid of the Google Translate bit was the important part 😄